### PR TITLE
add_camp added

### PR DESCRIPTION
**Pull request by souravsarangi**

### CODE REVIEW

File: app/views/camp/add_new.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        echo "Submission failed";
+    }
+    echo "<br><br>";
+}
+?>
+<form action="" method="POST">
+    
+        <legend>Basic details about the Camp</legend>
+        <label for="name">Name: </label><input id="name" name="name" required /><br>
+        <label for="organisation_id">Organisation: </label><input id="organisation_id" name="organisation_id" required /><br/>
```
Comment (gauravmittal1995) [inline app/views/camp/add_new.php:18]: We can't use organisation id directly. 
@nisargjhaveri We seriously need to create a auto complete widget. Thoughts??

File: app/views/camp/add_new.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          <!--  <select id="organisation" name="camp[organisation]" required>
+                <option value="" disabled selected>Select Organisation</option>
+                <option value="F">Female</option>
+                <option value="M">Male</option>
+                <option value="O">Other</option>
+            </select><br>-->
+        <label for="camp_head">Camp head: <input id="camp_head" name="camp_head" required /></label><br/>
```
Comment (nisargjhaveri) [inline app/views/camp/add_new.php:25]: While adding camp, we also need to add head details. Not just ID.

File: app/views/camp/add_new.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            <!--<select id="camp_head" name="camp[camp_head]" required>
+                <option value="" disabled selected>Select Head</option>
+                <option value="F">Female</option>
+                <option value="M">Male</option>
+                <option value="O">Other</option>
+            </select><br>-->
+        <label for="population">Population: </label><input id="population" type="text" name="population" /><br>
+        <label for="voluteers">Volunteers: </label><input id="volunteers" type="text" name="volunteers" /><br>
+        <label for="status">Status: </label><input id="stauts" type="text" name="status" /><br>
+        <label for="contact_id">Contact_Id: </label><input id="contact_id" type="text" name="contact_id" /><br>
```
Comment (nisargjhaveri) [inline app/views/camp/add_new.php:35]: Again, we need to add the contact details, not just contact ID

File: app/views/camp/add_new.php (hunk 2 above)
Comment (souravsarangi) [inline app/views/camp/add_new.php:25]: Hey this PR need not be merged. Sandeep has already created it with the UI so add that one.

Comment (nisargjhaveri) [inline app/views/camp/add_new.php:25]: That one has bigger problems.. :/ So I thought we'll fix this and merge it. Anyway, your choice.

Comment (souravsarangi) [inline app/views/camp/add_new.php:25]: okay on it

**Pull request closed without merging**